tests/test_csc.py

```diff
@@ -247,24 +276,29 @@ def m2_close_callback(data):
                                                                                      timeout=timeout)
 
                 # Send applyCorrection command
+                logger.debug('Send applyCorrection command')
                 azimuth = np.random.uniform(0., 360.)
                 # make sure it is never zero because np.random.uniform is [min, max)
                 elevation = 90. - np.random.uniform(0., 90.)
 
-                async def publish_mountEnconders(ntimes=5):
+                async def publish_mountEncoders(ntimes=5):
 
                     for i in range(ntimes):
-                        _azimuth = np.zeros(100)+azimuth
+                        _azimuth = np.zeros(100) + azimuth
                         # make sure it is never zero because np.random.uniform is [min, max)
-                        _elevation = np.zeros(100)+elevation
+                        _elevation = np.zeros(100) + elevation
                         harness.atmcs.tel_mount_AzEl_Encoders.set_put(
                             azimuthCalculatedAngle=_azimuth,
                             elevationCalculatedAngle=_elevation)
                         await asyncio.sleep(salobj.base_csc.HEARTBEAT_INTERVAL)
 
-                await publish_mountEnconders()
+                await publish_mountEncoders()
+
+                # FIXME: WHY DOESN'T THIS WORK? Related to function above
```

Comment:
What exactly is not working? did you managed to make it work? can you remove this?

Reply:
if `publish_mountEncoders` is a class method you should call it as `self.publish_mountEncoders`. But, as I explained above, I don't think this is really what you want to do.

Reply:
fixed.